client/address.c

```diff
@@ -7,7 +7,7 @@
 static const uint8_t bits2mime[64] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
 static uint8_t mime2bits[256];
 
-//intialization of addresses module
+/* intializates the addresses module */
```

Comment:
// initializes the address module

Reply:
Sorry, type. I wrote "initization" first, but I decided to change it to "initializes" later.